packages/sdk-ark/source/client.service.test.ts

```diff
@@ -72,27 +72,6 @@ describe("ClientService", () => {
 				expect(result).toBeObject();
 				expect(result.items()[0]).toBeInstanceOf(ConfirmedTransactionData);
 			});
-
-			it("multiple addresses", async () => {
```

Comment:
ARK supports search by multiple identifiers so why would this feature be dropped? It's the coins responsibility to ensure consistent behaviour internally, meaning something like that doesn't support it should ensure that only the first address is looked up but with ARK its possible to request data for many addresses at once to reduce network overhead.

Reply:
i thought because this feature isn't used in DW, it's safe to be dropped
but I think the SDK doesn't just belong to DW
i'll revert it

Reply:
The SDK is completely detached from DW and used in other products too. It adds features that DW needs if they are general purpose enough that other products also need them.